scripts/west_commands/pygit2_helpers.py

```diff
@@ -230,7 +230,7 @@ def _invert_keys_val_list(kvs: list[tuple[str, list[str]]],
 # Keep its definition sorted alphabetically by key.
 _AREA_TO_SHORTLOG_RES = [
     ('Arches', ['arch(/.*)?', 'arc(/.*)?', 'arm(/.*)?', 'esp32(/.*)?',
-                'imx(/.*)?', 'native(/.*)?', 'native_posix', 'nios2(/.*)?',
+                'imx(/.*)?', 'native(/.*)?', 'native_sim', 'nios2(/.*)?',
```

Comment:
I don't think this needs a change. @aescolar?

Reply:
This is the first time I see this west command, and I do not even know what it is meant to do. So I can't really help here :(

Reply:
perhaps it should be added with `native_sim` in addition, ref: https://github.com/nrfconnect/sdk-nrf/commit/fff95136065278015530dde074a4602b8ee1d30d

although i'm in doubt if anyone actually is using this anymore to track important upstream changes:
> I used to use this for making blog posts about what changed in Zephyr.
We want to start including significant Zephyr changes in the NCS
release notes as well, so it will hopefully be useful for splitting up
the work among maintainers during upmerges.